src/test/java/com/thirdcc/webapp/web/rest/EventAttendeeResourceIT.java

```diff
@@ -133,13 +219,133 @@ public void createEventAttendee() throws Exception {
         List<EventAttendee> eventAttendeeList = eventAttendeeRepository.findAll();
         assertThat(eventAttendeeList).hasSize(databaseSizeBeforeCreate + 1);
         EventAttendee testEventAttendee = eventAttendeeList.get(eventAttendeeList.size() - 1);
-        assertThat(testEventAttendee.getUserId()).isEqualTo(DEFAULT_USER_ID);
-        assertThat(testEventAttendee.getEvenId()).isEqualTo(DEFAULT_EVEN_ID);
+        assertThat(testEventAttendee.getUserId()).isEqualTo(user.getId());
+        assertThat(testEventAttendee.getEventId()).isEqualTo(savedEvent.getId());
         assertThat(testEventAttendee.isProvideTransport()).isEqualTo(DEFAULT_PROVIDE_TRANSPORT);
     }
 
     @Test
-    @Transactional
+    public void createEventAttendee_WithNonExistingUserId_ShouldThrow400() throws Exception {
+        Event savedEvent = initEventDB();
+        int databaseSizeBeforeCreate = eventAttendeeRepository.findAll().size();
+
+        // Create the EventAttendee
+        EventAttendeeDTO eventAttendeeDTO = createDefaultEventAttendeeDTO();
+        eventAttendeeDTO.setUserId(NON_EXIST_USER_ID);
```

Comment:
actually, there is a more convenient way to write this ~
```
eventAttendeeDTO.setUserId(Long.MAX_VALUE);
```

Reply:
Noted, thanks for the suggestion